app/views/layouts/application.html.erb

```diff
@@ -58,32 +58,18 @@
           </div>
         </div>
       </div>
+      <% if @posts %>
       <div class="row lead-imgs">
+          <% @posts.first(4).each do |post| %>
```

Comment:
It'd be even better to limit it to four at the controller level. That way you're not carrying around all the posts in the system in that `@posts` instance variable.